modules/standard/FileSystem.chpl

```diff
@@ -498,6 +498,16 @@ proc symlink(oldName: string, newName: string) {
   if err != ENOERR then ioerror(err, "in symlink " + oldName, newName);
 }
 
+/* Sets the file creation mask of the current process to mask, and returns
+   the previous value of the file creation mask.
+   mask: the file creation mask to use now.
+*/
+proc umask(mask: int): int {
```

Comment:
Yes, but I don't think it's portable.  http://www.gnu.org/software/libc/manual/html_node/Setting-Permissions.html mentions getumask, which only returns the current value, but it's description says "Return the current value of the file creation mask for the current process. This function is a GNU extension and is only available on GNU/Hurd systems."